dowhy/causal_estimator.py

```diff
@@ -126,6 +182,48 @@ def do(self, x):
     def construct_symbolic_estimator(self, estimand):
         raise NotImplementedError
 
+    def get_confidence_intervals(self, estimate):
+        '''
+            Find the confidence intervals corresponding to any estimator
+            This is done with the help of bootstrapped confidence intervals
+
+            For more details, refer to the following link:
+            https://ocw.mit.edu/courses/mathematics/18-05-introduction-to-probability-and-statistics-spring-2014/readings/MIT18_05S14_Reading24.pdf
+        '''
+
+        # The array that stores the results of all estimations
+        simulation_results = np.zeros(self.num_ci_simulations)
+
+        # Find the sample size the praportion with the population size
+        sample_size = int( self.sample_size * len(self._data) )
+
+        if sample_size > len(self._data):
+            self.logger.warning("WARN: The sample size is greater than the population being sampled")
+        
+        self.logger.info("INFO: The sample size: {}".format(sample_size) )
+        self.logger.info("INFO: The number of simulations: {}".format(self.num_ci_simulations) )
+        
+        # Perform the set number of simulations
+        for index in range(self.num_ci_simulations):
+            new_data = resample(self._data,n_samples=sample_size)
+
+            new_estimator = CausalEstimator.get_estimator_object(new_data, self._target_estimand, self._estimate)
+            new_effect = new_estimator.estimate_effect()
+            simulation_results[index] = new_effect.value
+        
+        # Now use the data obtained from the simulations to get the value of the confidence estimates
+        # Sort the simulations
+        simulation_results.sort()
+        # Now we take the 5th and the 95th values
+        lower_bound_index = int( 0.05 * len(simulation_results) ) 
```

Comment:
can this also be a parameter? I think that's better in case someone wants a 99% confidence interval, or any other CI.

Reply:
Meant the constant 0.05, typically called the confidence_level